sonar-db/src/main/java/org/sonar/db/version/v61/AddErrorColumnsToCeActivity.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.version.v61;
+
+import java.sql.SQLException;
+import org.sonar.db.Database;
+import org.sonar.db.version.AddColumnsBuilder;
+import org.sonar.db.version.DdlChange;
+
+import static org.sonar.db.version.ClobColumnDef.newClobColumnDefBuilder;
+import static org.sonar.db.version.VarcharColumnDef.newVarcharColumnDefBuilder;
+
+public class AddErrorColumnsToCeActivity extends DdlChange {
+
+  private static final String TABLE_CE_ACTIVITY = "ce_activity";
+
+  public AddErrorColumnsToCeActivity(Database db) {
+    super(db);
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    context.execute(new AddColumnsBuilder(getDatabase().getDialect(), TABLE_CE_ACTIVITY)
+      .addColumn(newVarcharColumnDefBuilder().setColumnName("error_message").setLimit(1000).setIsNullable(true).build())
```

Comment:
Indeed, you need to verify the size when inserting the value.
I said 1000 seems not enough for me, because the first test I've tried (analyzing a project that contains a module that I've first analyzed) returns me a message of 400 characters. 
As ValidateProjectStep concatenate all errors that happen, I can imagine than 1000 will not be enough.